scripts/vmbuild.rb

```diff
@@ -109,49 +111,44 @@ def verify_run(output)
 Dir.chdir(IMGFAC_DIR) do
   targets.sort.reverse.each do |target|
     imgfac_target = target.imagefactory_type
-    vhd_image     = target.file_extension == "vhd"
+    ova_format    = target.ova_format
     $log.info "Building for #{target}:"
 
     tdl_name = imgfac_target == "azure" ? "base_azure.tdl" : "base.tdl"
```

Comment:
@garrett This will need to be changed as the value of `imagefactory_type` is changed for azure in this PR.  We could have just checked for `target` anyway..